x-pack/plugins/apm/server/routes/service_groups/get_services_counts.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ProcessorEvent } from '@kbn/observability-plugin/common';
+import { rangeQuery, kqlQuery } from '@kbn/observability-plugin/server';
+import { Setup } from '../../lib/helpers/setup_request';
+import { SERVICE_NAME } from '../../../common/elasticsearch_fieldnames';
+
+export async function getServicesCounts({
+  setup,
+  kuery,
+  maxNumberOfServices,
+  start,
+  end,
+}: {
+  setup: Setup;
+  kuery: string;
+  maxNumberOfServices: number;
```

Comment:
This looks unused

Reply:
It is unused, I can add a size (service inventory and service map already have max size specified)